changelog.d/19921_missing_dns_record_types.enhancement.md

```diff
@@ -0,0 +1,3 @@
+Added support for more DNS record types (HINFO, CSYNC, OPT, DNSSEC CDS, DNSSEC CDNSKEY, DNSSEC KEY)
+
+authors: @esensar
```

Comment:
```suggestion
authors: esensar
```

The `@` should actually be omitted because it is appended later. We updated the CI check to look for this which is why it is currently failing.

Reply:
Thanks and sorry about that :D